### PR TITLE
Update docs for dashboard firmware `v0.6.0`

### PR DESCRIPTION
**Pull request by Hackerman342**

### CODE REVIEW

File: website/docs/apps/autoplot_app/README.md [hunk 1]
```diff
@@ -110,14 +110,15 @@ The control panel is accessed by clicking on the gear icon located in the upper
...
+Starting with dashboard firmware release
+[**amiga-dash-v0.5.0**](https://github.com/farm-ng/amiga-dev-kit/releases/tag/amiga-dash-v0.5.0),
+there is support for up to four H-Bridge devices.
+These devices can be either latched or momentarily controlled,
+and can optionally be coupled for synchronous operation.
 
 ![Screenshot from 2024-05-02 18-05-04](https://github.com/farm-ng/amiga-dev-kit/assets/133177230/ffb45918-ed86-497d-a72b-5892d75053ab)
 
-Up to four PTO Devices are supported and can be remotely selected for operation.
+Up to two PTO Devices are also supported and can be remotely selected for operation.
```
Comment (guilhermedemouraa) [inline website/docs/apps/autoplot_app/README.md:121]: I thought it was four. Did we intentionally decreased the number of supported PTOs?

**Review by guilhermedemouraa — APPROVED** [review]
LGTM

File: website/docs/apps/autoplot_app/README.md (hunk 1 above)
Comment (Hackerman342) [reply website/docs/apps/autoplot_app/README.md:121]: Only two are enabled in the firmware, to reduce the memory footprint

**Review by AgustinBRodriguez — COMMENTED** [review]
Looks good to me! Obrigado

**Pull request merged**